MyLibrary/MyLibraryApp.API/Controllers/BooksController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MyLibraryApp.Application.Models.Books;
+using MyLibraryApp.Application.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.API.Controllers
+{
+    [ApiController]
+    [Route("Books")]
+    public class BooksController :ControllerBase
```

Comment:
I would personally use `books`, `orders`, etc for API routes. I've seen camel case being widely used so I kind of got used to it.
Also, you need to add a whitespace between `:` and `ControllerBase` so that the code becomes easier to read.